app/src/main/java/com/example/cooktaildb/ui/MainActivity.kt

```diff
@@ -10,7 +12,8 @@ import com.example.cooktaildb.ui.filter_fragment.FilterFragment
 import com.example.cooktaildb.ui.home_fragment.HomeFragment
 import com.google.android.material.navigation.NavigationBarView
 
-class MainActivity : BaseActivity<ActivityMainBinding>(ActivityMainBinding::inflate), NavigationBarView.OnItemSelectedListener{
+class MainActivity : BaseActivity<ActivityMainBinding>(ActivityMainBinding::inflate),
+    NavigationBarView.OnItemSelectedListener {
```

Comment:
```suggestion

    NavigationBarView.OnItemSelectedListener {
```
thêm 1 blank line ở ngay trên hàm